README.md

```diff
@@ -2508,7 +2508,9 @@ IEEE Conf. Computer Vision and Pattern Recognition Workshops (CVPRW), 2021. [Pro
 - <a name="Joubert21fins"></a>Joubert, C., Marcireau, A., Ralph, N., Jolley, A., van Schaik, A., Cohen, G.,  
 [Event Camera Simulator Improvements via Characterized Parameters](https://www.frontiersin.org/articles/10.3389/fnins.2021.702765/full)  
   Front. Neurosci., 2021, [IEBCS simulator](https://github.com/neuromorphicsystems/IEBCS)
-
+- <a name="Lin2022dvs"></a>Lin, S., Ma, Y., Guo, Z. and Wen, B.,
```

Comment:
key: Lin22eccv (if it does not collide with another entry)

Reply:
The line breaks are still missing.

![image](https://github.com/uzh-rpg/event-based_vision_resources/assets/8024432/1f12c2a7-54cc-4e65-b15e-95186563ba21)

Authors, title and publication in three separate lines (3 lines in total), please.